tests/ai_safety/evalhub/test_evalhub_reconcile_observability.py

```
@@ -0,0 +1,1276 @@
+        Pod.get(
+            client=admin_client,
+            namespace=operator_ns,
+            label_selector="control-plane=controller-manager,app.kubernetes.io/name=trustyai-service-operator",
+        )
+    )
+    assert pods, "No operator pod found"
+    pod = pods[0]
+    response = requests.get(
+        f"https://{pod.instance.status.podIP}:{OPERATOR_METRICS_PORT}/metrics",
+        headers={"Authorization": f"Bearer {operator_metrics_token}"},
+        verify=False,
```

Ruff (0.16.1)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)